linux/libdriver/base/keccak.h

```diff
@@ -0,0 +1,131 @@
+/** @file
+ * @defgroup libevm_keccak keccak
+ * Keccak 256 digest
```

Comment:
It should be noted that this implementation does not follow the FIPS-202 based standard (aka finalized NIST SHA-3), it follows the Ethereum Keccak-256.